### PR TITLE
Add Markdown support

### LINKED ISSUE #86: Implement all the Pages for the frontend.
Since the  **Header**, **Footer**, and the **Entity Component** are almost implemented and the basic setup is done.
It is time to implement the various pages so that further work can be done on each one.
image

Steps to implement:
- To create new pages for each of the `Contribute`, `Projects`, `Committee`, `Chapters`
- To fetch the API data from the backend, show the entity card on each page.

Related issues:
  - #88 
  - #89 
  - #90 
  - #91

### LINKED ISSUE #251: Add Markdown support
**Is your feature request related to a problem? Please describe.**
Currently raw markdown is displayed as it is.
**Describe the solution you'd like**
To introduce Markdown support for displaying markdown content on frontend using `React Markdown` and `remark-gfm` plugins.

### PR DESCRIPTION
**Pull request by harsh3dev**
Resolves #251 
Resolves #86 

Added the markdown support using `markdown-it`

### CODE REVIEW

**Comment by harsh3dev** [comment]
https://github.com/user-attachments/assets/09fc2978-a1e7-467a-a651-5499c32225c7

File: frontend/src/index.css [hunk 1]
```diff
@@ -116,3 +116,7 @@ a {
 .animate-scaleIn {
   animation: scaleIn 0.3s ease-out;
 }
+
+ol {
+  list-style: decimal;
+}
```
Comment (arkid15r) [inline frontend/src/index.css]: Could you narrow it down to markdown scope only?

File: frontend/src/components/Card.tsx [hunk 2]
```diff
@@ -2,15 +2,15 @@ import { FontAwesomeIcon, FontAwesomeIconProps } from '@fortawesome/react-fontaw
...
-import { CardProps, tooltipStyle } from 'lib/constants'
-import FontAwesomeIconWrapper from 'lib/FontAwesomeIconWrapper'
-import { cn } from 'lib/utils'
-
-import ActionButton from 'components/ActionButton'
-import ContributorAvatar from 'components/ContributorAvatar'
-import { Icons } from 'components/data'
-import DisplayIcon from 'components/DisplayIcon'
-import TopicBadge from 'components/TopicBadge'
+import ActionButton from './ActionButton'
```
Comment (arkid15r) [inline frontend/src/components/Card.tsx]: Please use absolute imports

**Review by arkid15r — CHANGES_REQUESTED** [review]
Looks good, great work!

Could you add some basic tests for the new functionality?

Thank you!

**Review by arkid15r — APPROVED** [review]
LGTM 👏

**Pull request merged**